virtcontainers/qemu.go

```diff
@@ -638,6 +638,11 @@ func (q *qemu) setupVirtiofsd() (err error) {
 	var listener *net.UnixListener
 	var fd *os.File
 
+	_, err = os.Stat(q.config.VirtioFSDaemon)
+	if err != nil {
+		return err
```

Comment:
might be worth making this error a little bit more verbose .. ?

```go
return fmt.Error("No such virtio fs daemon: %v", err)
```

Reply:
In the new version, I went for the same message used when the hypervisor is not found.
Thanks for the suggestion!